### DISCUSSION TITLE
Slingbox ProHD PC Unbricking-Setup-Network-Guide

### DISCUSSION THREAD

**Discussion by proudx2** [GerryDazoo/Slinger#357] [General]
https://www.mediafire.com/file/ppw6ep09t99xhu0/Slingbox_ProHD_PC_Unbricking-Setup-Network-Guide.pdf/file

**Comment by PeteSki1618** [comment]
Just started reading the directions and before I start, it says to block certain outgoing ports.  I'm not too sure how to do that but will figure it out.  However, once done and assuming it worked, will I be able to view my Slingbox remotely since the ports are blocked?  Also, I have port 8181 open so I can get onto my router remotely and reboot it in case of any problems with my cameras and other security devices while I'm away.  Can I leave 8181 open while doing this setup or does that get closed as well?  Can I reopen it once the Slingbox is working?  Thanks in advance for any help!

**Reply by proudx2** [reply]
You are only blocking outgoing from your slingbox to the wan/internet. This will not effect any incoming port mappings like your 8181 you have.
 
While we do give specific tcp/udp port numbers to block for the novice we recommend blocking all outbound communication from the slingbox  tcp/udp ports 1 - 65535 in 1 simple rule.

**Reply by PeteSki1618** [reply]
Ok, great.  Thank you.  I will try to figure out how to do that.

**Reply by Mikoshi-nyudo** [reply]
With firewalls the order (priority) of the rules are important. So as an explanation of a general concept, you can a have a total block for all incoming and outgoing traffic (TCP/UDP) from the SB IP to anywhere through the WAN interface and then have only a single IP allowed in and out to it, placed before or above that blanket block/drop rule. This is an easy setup method for a static IP setup from home when accessing a remote box over WAN, not so much for a dynamic IP.

The same concept also applies for rules based on ports too. You can block an entire range i.e 1-65535, and prior to it have a single rule to allow just the incoming remote viewing port. Different firewalls have different terminology and some require pre-defined services for these things that you use, others allow you to just enter values and protocols directly. But overall the general concept is similar. 

The guide is not intended to cover that aspect, other than a concept, as there are just too many variables to cover.. and if someone is having some issue with a specific Firewall it would be better to post something on discord, and other users that have the brand or model might be able to assist those having such difficulties.

**Reply by PeteSki1618** [reply]
Thanks for your reply.  I have an M1 that is working.  Just to "try" messing with the ports, I blocked all outgoing ports and the M1 stopped working.  So, before I mess with the ProHD, if the M1 won't work with them all blocked, I'm assuming the PHD won't either.  Am I missing something?

**Reply by GerryDazoo** [reply]
Yes, The M1 and ProHD should work with all outgoing traffic blocked. I know mine does. You've got something else wrong.

**Reply by PeteSki1618** [reply]
Once again, I really want to thank you for putting in all the time and effort to help us all out and for dealing with the stupidity of novices like myself.  A lot of this is WAY above my computer knowledge.  I have used Slingbox for many, many years to remotely watch my home TV from our small vacation home and this has been a real inconvenience.  The M1 is not mine and I have to give it back once I get my ProHD's going (if ever) so I'm still trying.  So please bear with me during this ordeal but know you are very much appreciated, along with all of the others that have been so helpful to get me to this point so far.

So here is what I did and, if you don't mind, please let me know what I am doing wrong.  With my M1 running on a Win10 computer, I went into Control Panel, Windows Defender Firewall, Advanced, Outbound Rule, New Rule.  I selected Port, All Remote Ports, Block the Connection, then on the last screen with the choices of Domain, Private or Public, I selected ONLY Public (I think I tried it with all 3, too) and as soon as I activated the rule, the server stopped.  Not sure if this matters but in the config.ini I have the M1 on port 5301 and the server on port 8181.  My router has port 8181 open with Enable HTTPS Server and Enable Remote Management so I can access my router remotely in case of any problems while I'm away.  

Thanks in advance once again!

**Reply by PeteSki1618** [reply]
I forgot to mention that I also have Port Forwarding on the M1 at TCP UDP both 5301 and Virtual Server at External 9176 Internal 8181.  Not sure if that has anything to do with it but it is working on the M1.

**Reply by proudx2** [reply]
You don’t want to block outgoing ports from you windows pcs defender pc firewall so undo that.  You need to setup the block on your internet router/firewall from your sling box prohd not your pc.

**Reply by PeteSki1618** [reply]
Oh, sorry.  Again, being a novice I looked up how to block ports and that is what I found.  So, I have a DLink router.  When I go into Firewall it has IPv4 and IPv6 rules and there are also a whole bunch of other settings which are Disabled.  If I go into either the 4 or 6 rules it asks for a Name, Source IP Range and WAN or LAN, Destination IP Range and WAN or LAN, and Port Range either TCP, UDP or Any.  Is that where I go to block the port?  I realize I am blocking the port so it doesn't brick itself again, but if I ever get it running, don't I have to unblock the port so I can access it remotely and won't it brick itself again?  Sorry for my stupidity and THANK YOU so much for your patience and help.

**Reply by sp222222** [reply]
read what's written before..... yes, take the IP4 rules and block all ports to/from the slingbox itself. THEN use port forwarding rule on the dlink router to open up one port for the slinGER server itself to get through from the outside. 
my asus takes four rules to make a single slingbox "blocked from WAN"; yours might take one. You are not closing 8181 or any other service. this rule is to block to/from of the IP the slingbox itself uses (and this IP should be static and not DHCP). And unless you plan on not accessing the router and slingbox externally at the same time, I strongly suggest you modify the configM1.ini file to use another port (8181) for the slinGER server. the 5301 will remain the same in that configM1.ini file.

**Reply by PeteSki1618** [reply]
Once again, thanks for your input and sorry for my lack of knowledge.  Good news is that I unplugged the cable from my router to the modem to block outside access and went through the instructions, unbricked the ProHD and it worked perfectly.  Before I plugged the WAN back in I went to the router and turned on Parental Controls, hoping that would block it from going out onto the internet and bricking but it didn't work at all (but didn't rebrick itself).  I am now going into the Firewall settings, IPv4 rules, I have the option to "Turn on IPv4 rules and DENY Rules listed" which I assume is what I need to do, and now attempting to put in a rule to block outgoing traffic and that is where I am confused.  I have attached a screenshot and, if that is where I'm supposed to be, what values should I put in for this new Rule?  The box is at 192.168.0.253 and port 5001.  I have port forwarding on so that I can access it remotely.  Am I in the right setup to block the outgoing traffic?  Thank you again for your continued support and patience!!!

![IPv4 rules](https://user-images.githubusercontent.com/117535328/204166212-7da2dfe1-c9fe-47c0-9bd0-d7312ca451bc.jpg)
![Firewall](https://user-images.githubusercontent.com/117535328/204166003-72cb30c6-520f-45a4-9fc7-18146c600b75.jpg)
![Port forwarding](https://user-images.githubusercontent.com/117535328/204166010-48f51054-b0a7-427e-bc66-cbab4cef1d5e.jpg)

**Reply by PeteSki1618** [reply]
I know most of you have given up on me but I'm still trying to block the outgoing ports so I don't rebrick the ProHD.  
As suggested by others, I blocked all out going ports by choosing NEVER for the Schedule in my Port Forwarding Rule.  However, it says that I have Overlapping Ports.  Is it because I have some ports already forwarded?  I need 9176 for the server, 81 for a camera and 5001 and 5301 for the ProHD and M1.  I hope someone can get me past this last hurtle.  Thanks again for your help.

![Overlapping Ports](https://user-images.githubusercontent.com/117535328/205161168-2353ba7a-9fcb-4e06-8438-2b14a7899996.jpg)



![Port Forward 2](https://user-images.githubusercontent.com/117535328/205163066-1fa8782d-73a5-41f0-8324-4d22a7e20d52.jpg)
![Virtual Server 2](https://user-images.githubusercontent.com/117535328/205163084-9c6a5595-8a10-4b32-a016-2e8dc7ae2e3b.jpg)

**Reply by proudx2** [reply]
You don’t do a port forwarding rule to block outgoing communication from slingbox.  Port forwarding configuration is for incoming communication on a specific internet port tied to a specific local network port.  

To block the slingbox outgoing communication you do this from the ip4 rules under your firewall. Set source lan and ip as your slingbox snd for destination select wan. For ports do tcp 1-65535 and udp do same. If it will net let you specify port range do  5678 only as that single port will stop it from bricking as well.

**Reply by PeteSki1618** [reply]
Thank you SO much for your reply.  This is driving me nuts.  One last question (I hope): 
 Is this what it should look like?  Thanks again!!!
![IPv4 Rule](https://user-images.githubusercontent.com/117535328/205183825-f02a063f-f3eb-4907-b022-4816248a0caa.jpg)

**Reply by edutsilva** [reply]
> Thank you SO much for your reply. This is driving me nuts. One last question (I hope): Is this what it should look like? Thanks again!!! ![IPv4 Rule](https://user-images.githubusercontent.com/117535328/205183825-f02a063f-f3eb-4907-b022-4816248a0caa.jpg)

So, if this rule is not working, try change it to:
Name: BlockSlingbox
Source IP Address Range: LAN 192.168.0.253
Destination IP Adress Range : Wan 0.0.0.0 (the destination can´t be a LAN IP. It has to be a WAN IP. In this case, 0.0.0.0 is ALL WAN Ips)
Port Range: Any
Schedule: Always

I believe it will work. If it does not, we will try something else in the Destination IP field.

**Reply by PeteSki1618** [reply]
Just tried it but it says to enter a valid range like 1.1.1.1 - 1.1.1.2.
Thanks for your suggestion though.
Not to sound even more stupid, in one of the other posts it was suggested to list a number of ports and they gave some examples of ports to try and specifically one for the bricking (can't find the post right now).  Maybe I'm in the wrong place (I'm under Firewall, IPv4 Rules) where they are requiring an IP address and not a specific port?  So confused!!!

**Reply by Toei79** [reply]
Peter share screen of router the port thing maybe its different  on yours.

Mine its asus, but not check this yet

**Reply by PeteSki1618** [reply]
Thanks in advance for trying to help.  Mine is a DLink Covr-3902.
Here's the manual:
https://www.bing.com/ck/a?!&&p=6b899fd9e2a8bdc5JmltdHM9MTY2OTkzOTIwMCZpZ3VpZD0zNjQzNmUzOS1lZmU2LTYxOWEtMzY4Yy02NWQyZWJlNjYyMjYmaW5zaWQ9NTE2Ng&ptn=3&hsh=3&fclid=36436e39-efe6-619a-368c-65d2ebe66226&psq=COVR-3902_REVA_MANUAL_v1.01_WW_EN.pdf&u=a1aHR0cHM6Ly9zdXBwb3J0LmRsaW5rLmNvbS9yZXNvdXJjZS9wcm9kdWN0cy9jb3ZyLTM5MDItdXMvcmV2YS9DT1ZSLTM5MDJfUkVWQV9NQU5VQUxfdjEuMDBfV1dfRU4ucGRm&ntb=1

**Reply by proudx2** [reply]
> Just tried it but it says to enter a valid range like 1.1.1.1 - 1.1.1.2.
> Thanks for your suggestion though.
> Not to sound even more stupid, in one of the other posts it was suggested to list a number of ports and they gave some examples of ports to try and specifically one for the bricking (can't find the post right now). Maybe I'm in the wrong place (I'm under Firewall, IPv4 Rules) where they are requiring an IP address and not a specific port? So confused!!!

Try 0.0.0.1-255.255.255.254

**Reply by PeteSki1618** [reply]
Thanks for your suggestion. It doesn't allow me to enter that either. I then went back to Port Forwarding and put in the following ports to NEVER allow access to:
5678,5680,3722,3490,3074,80,8080,443 and after unbricking it (by physically unplugging my router from my modem), the Server software finds it perfectly. But as soon as I try to access it remotely, it instantly bricks itself again. Here's what I tried. Any other
![PortBlocked](https://user-images.githubusercontent.com/117535328/205509467-a88b308f-0ad1-41c2-90fa-9bc3f2818c80.jpg)
 suggestions would be greatly appreciated!

**Reply by proudx2** [reply]
Port forwarding is not what you want so remove those rules. I think your best alternative is to set a static ip with gateway of 127.0.0.1. start over with internet unplugged and follow guide completely except ignore the port blocking prerequisites. Once setup continue to keep internet unplugged and  follow the optional setup static ip network configuration.  For the ip you can set to the dhcp reservation ip you had. For gateway use 127.0.0.1. This will prevent the slingbox from talking to the internet.  Make sure to follow the directions completely in the correct order.  This should work.

**Reply by PeteSki1618** [reply]
Thank you for your patience.  I am a novice when it comes to this stuff and am trying my best.  I'm not sure where you want me to put in those numbers.  The above is what Port Forwarding and Virtual Server shows.  The other place I was messing with was under Firewall and IPv4 Rules.  I previously posted what options I had there but could not put in a destination IP range.  I have the ProHD with a reservation of 192.168.0.253.  Is that the static ip?  Are you suggesting putting in 127.0.0.1 instead?  Is that different than the Gateway (don't see a separate entry)?  I'm so sorry about all of the questions but I am trying my best with limited knowledge.  Thanks again.

**Reply by proudx2** [reply]
So your going to not use your dlink router t do any configuration.  Your only going to unplug your routers internet connection and perform entire configuration using the slingbox software setup a static IP address on the slingbox itself with a gateway of 127.0.0.1. Yes you can use same 192.168.0.253 as static ip but will use 127.0.0.1 for gateway. This method will block slingbox from talking to internet and getting bricked. Once setup you can plug internet back in. 

Again follow entire guide closely  in correct order following all steps and go into optional network configuration to do your static ip setup at which point you will use the 127.0.0.1 as the gateway

**Reply by edutsilva** [reply]
> Thanks for your suggestion. It doesn't allow me to enter that either. I then went back to Port Forwarding and put in the following ports to NEVER allow access to: 5678,5680,3722,3490,3074,80,8080,443 and after unbricking it (by physically unplugging my router from my modem), the Server software finds it perfectly. But as soon as I try to access it remotely, it instantly bricks itself again. Here's what I tried. Any other ![PortBlocked](https://user-images.githubusercontent.com/117535328/205509467-a88b308f-0ad1-41c2-90fa-9bc3f2818c80.jpg) suggestions would be greatly appreciated!

Port forwarding is an inbound feature. It only works for outsite to inside traffic, so it won´t work to block inside to outside traffic.

Have you tried placing your rounter IP as destination IP? If not, you should try it. It´s the IP you use to admin it.
Good luck.

**Reply by PeteSki1618** [reply]
Thanks Proudx2 for your suggestion.  I'm going to try the suggestion from Edutsilva first since that seems easier.  However, Edutsilva, if I create an IPv4 rule in my Firewall, for the Destination IP are you suggesting 192.168.0.1 (router login) and set it also to WAN as shown below or use the WAN IP address?  If using the WAN IP address and my IP changes due to DCHP, that would mean I have to change it whenever my IP changes, right?  Thanks for your help.  I'll wait for your reply before trying it so I don't brick it again by doing the wrong thing.  
![IPv4 rule 4](https://user-images.githubusercontent.com/117535328/205703795-800116bb-b439-4c7a-8d47-b85ed46bb284.jpg)

**Reply by PeteSki1618** [reply]
Success!!!  Halleluiah!  Thank you to ALL of you for your patience and expertise.  I was able to get it to work using Proudx2's suggestion of a static IP.  I will probably be back with a few more questions because my remote control is not working correctly but that is minor.  And, since I have both an M1 and ProHD, when I get some time I may try to get them both working simultaneously (or just count my blessings that either one works).   For now, just VERY happy to have it working again. Thanks again to ALL!!!

**Reply by DoogieHowitzerWMD** [reply]
> Success!!! Halleluiah! Thank you to ALL of you for your patience and expertise. I was able to get it to work using Proudx2's suggestion of a static IP. I will probably be back with a few more questions because my remote control is not working correctly but that is minor. And, since I have both an M1 and ProHD, when I get some time I may try to get them both working simultaneously (or just count my blessings that either one works). For now, just VERY happy to have it working again. Thanks again to ALL!!!

Congrats, that's awesome!

If you managed to get the ProHD running - blocking it off from the internet, unbricking it, reconfiguring it, and getting it up and running with Slinger - simply adding the M1 to the mix should not be too difficult of a task.

**Reply by edutsilva** [reply]
Use the Slingplayer to change the remote control used by your slingbox.

**Reply by PeteSki1618** [reply]
I was afraid to go back into Slingplayer and mess up the configuration.  If I do, use the one WITHOUT the blue house, right?  Can I just go to the Audio Settings which is where I think I remember the remote was located?  It's weird because the remote works perfectly with the M1 (I had to change a few of the buttons around to match my cable box).  But, using the exact same profile on the ProHD works but the buttons don't react the same way.  Again, not that important right now.  Thanks again for everything.

**Reply by proudx2** [reply]
You can safely go back into slingplayer just don’t run network configuration again. You can rerun video/ audio configuration  setup and reconfigure as needed. Yes use the direct connect shortcut without blue house.

**Reply by PeteSki1618** [reply]
Ok thanks.  I'll give it a try when I get a chance to play around a bit more.  Just so happy to have it up and running and not bricking anymore.

**Comment by RandomMan719** [comment]
If anyone uses this guide on Windows 11, could you post and let us know if it works or if there are issues on Windows11.  Thanks.

**Reply by pfuller88** [reply]
i had no issues with windows 11 and unbricking a PRO model.   On my windows 11 PC I ran the slinger program in compatibility mode, with windows 7 chosen.

**Reply by GerryDazoo** [reply]
When you say "slinger"  do you mean my code, slingbox_server?. There should be no reason to run it in Windows 7 compatibility mode.

**Reply by pfuller88** [reply]
Sorry,  I meant the Slingbox 1.5 US software.  I followed the directions in another post, used windows 11.

**Reply by RandomMan719** [reply]
@pfuller88 Thanks for letting us know, the Unbricking process works on Windows 11, good to know.

@pfuller88 I also want to clarify that you unbricked a **_"ProHD"_** model, not a **_"Pro"_** model (pro unbricking hasn't been done, as far as I know), correct?  You might want to edit that post, so you don't confuse people, thank you.

**Reply by pfuller88** [reply]
ProHD model.

**Reply by Toei79** [reply]
feels like its some kind of temporal block by sling , while you not apply this ''remedy''  so its not so bricked?

**Comment by edutsilva** [comment]
Hi.
First of all, after the reset, my ProHd horseshoe is not rock solid red. It´s slowly kind of blinking.
Then, I´m getting an error when trying to set new passwords: "Operation Failed Error: 0x92340070". What am I doing wrong?
thank you.


![sling_error](https://user-images.githubusercontent.com/116834607/205181066-bcc47873-ace3-460a-9ce5-b50860c9e147.png)

**Reply by proudx2** [reply]
That’s a bricked message.  Your  not resetting or you do not have outbound blocking rules in place from your slingbox. Start over by unplugging internet and resetting slingbox snd checking or recreating dhcp reservation  and  blocking rules from  the slingbox

**Reply by edutsilva** [reply]
Thanx. Now it worked. One last thing. I´m having trouble with the remote control after the factory reset. It was working before when the PROHD was not bricked (before the servers shutdown).
Side note: My other slingbox (M1) never got bricked and the remote control works fine. It uses the same IR codes that the ProHD uses.
Thank you.

**Reply by proudx2** [reply]
What device snd codes?  Do you have the Custom bin file in the sbav directory for it?  Alternatively Did you try using a native device in database?  Make sure sbav-jb.sav is in the sbav folder as well or you will net get device selection in setup

**Reply by edutsilva** [reply]
Thanks for replying. The sbav-jb,sav is in C:\Program Files (x86)\Sling Media\SlingPlayer\SBAV
I am from Brazil. Like I said, the remote control is working for my other box (M1) and It worked for the ProHd before (using slinger) it got bricked. So chances are that the unbrickig caused this issue.
When I try to use the Slingplayer APP, I can locate what is supposed to be the right controller, but it does not work. I checked the infrared emitter with my cell phone and it is sending the commands to the cable box, but it is not responding. 
Any ideias?

**Reply by bradut** [reply]
@edutsilva I un-bricked my PRO-HD and **got the IR Remote working** yesterday, so my memories are fresh.
- I had the same experience with the initial error message. The SlingBox got bricked back in seconds if not properly protected.
- IR Remote setup:
   - I ran the original setup with both the European and North American versions of SlingPlayer 1.5.x. The European one allows selecting the video source + the cable box, but my model was not listed, and after I tried 15 ... 20 options (out of 50 I think) I gave up.  
   Note: I ran SP 1.5.x on some Virtual Machines to avoid overriding the original log files and any associated info from SP 2.0.4 I used to run on my machine.
  
    - to get the IR Remote to work, I opened my SlingPlayer 2.0.4.522 which I was using before 9 Nov.2022, selected the un-bricked SlingBox (I gave it the same name and password during the post-brick setup with SP 1.5.x, just in case), and ran the Audio/Video setup, which allowed me to select my Cable box model and my channels.  By using SP 2.0.4 I risked being bricked again, but **it worked**.
    Note: I initially tried this with the older SP 2.0.1?, but got bricked. 
  
Side note: After the Audio/Video setup I was able to watch from my SB on SlingPlayer for Android, but _only once_.  Any subsequent attempts didn't work: initially the SlingPlayer complained about SB old firmware, and after upgrading the firmware, got a weird message that "Slingplayer for Android is not compatible with your Slingbox"

**Comment by proudx2** [comment]
does it let you specify a wan destination range of nothing or all?  you need to block the entire wan outbound.  try leaving it blank or hit the drop down to see if you see any other options.
________________________________

**Reply by PeteSki1618** [reply]
The Destination IP Range drop down only has LAN or WAN.  It won't let me leave it blank.  The Port Range is TCP, UDP or ANY.  I don't know the syntax for putting in an IP range???  Thanks for your reply.

![IPv4 Rule2](https://user-images.githubusercontent.com/117535328/205211233-c3050a87-2a68-4307-8398-c22f647a3b34.jpg)

**Reply by proudx2** [reply]
What model is your  router?  I’d like to look at the online pdf for it to see the correct syntax for all wan ip destinations

**Reply by edutsilva** [reply]
In SCHEDULE', you cannot use NEVER. It means the rule you are creating will never be used. Change it to ALWAYS or something like that,

**Reply by PeteSki1618** [reply]
I was told that ALWAYS allows that port to be open and if I want to block the traffic to set it to NEVER.  
Here's the manual:
https://www.bing.com/ck/a?!&&p=6b899fd9e2a8bdc5JmltdHM9MTY2OTkzOTIwMCZpZ3VpZD0zNjQzNmUzOS1lZmU2LTYxOWEtMzY4Yy02NWQyZWJlNjYyMjYmaW5zaWQ9NTE2Ng&ptn=3&hsh=3&fclid=36436e39-efe6-619a-368c-65d2ebe66226&psq=COVR-3902_REVA_MANUAL_v1.01_WW_EN.pdf&u=a1aHR0cHM6Ly9zdXBwb3J0LmRsaW5rLmNvbS9yZXNvdXJjZS9wcm9kdWN0cy9jb3ZyLTM5MDItdXMvcmV2YS9DT1ZSLTM5MDJfUkVWQV9NQU5VQUxfdjEuMDBfV1dfRU4ucGRm&ntb=1

**Reply by edutsilva** [reply]
You were told wrong. In page 88 it says:

"Schedule: Use the drop-down menu to select the time schedule that the rule
will be enabled for. The schedule may be set to Always Enable, or
you can create your own schedules in the Schedules section. Refer
to Refer to Schedule on page 88 for more information."

If you add a rule that is scheduled to NEVER be enabled, it will NEVER work.

Best of luck.

Eduardo

**Reply by PeteSki1618** [reply]
Thank you so much for looking at it.  So, I'm totally confused how to block the outgoing traffic so the ProHD doesn't brick.  After so many suggestions from all of you great guys who are trying to help, I still can't block the traffic.  I got it unbricked and working but as soon as I put it online it rebricks.  I'd appreciate any insight you may have.  Thanks again.

**Reply by edutsilva** [reply]
I just answered your other post. The one with a router screen print.

**Reply by proudx2** [reply]
> The Destination IP Range drop down only has LAN or WAN. It won't let me leave it blank. The Port Range is TCP, UDP or ANY. I don't know the syntax for putting in an IP range??? Thanks for your reply.
> 
> ![IPv4 Rule2](https://user-images.githubusercontent.com/117535328/205211233-c3050a87-2a68-4307-8398-c22f647a3b34.jpg)

specify this range  for wan destination. 
0.0.0.1-255.255.255.254

**Reply by PeteSki1618** [reply]
Thanks for your suggestion.  It doesn't allow me to enter that either.  I then went back to Port Forwarding and put in the following ports to NEVER allow access to:
5678,5680,3722,3490,3074,80,8080,443 and after unbricking it (by physically unplugging my router from my modem), the Server software finds it perfectly.  But as soon as I try to access it remotely, it instantly bricks itself again.  Here's what I tried.  Any other suggestions would be greatly appreciated!
![PortBlocked](https://user-images.githubusercontent.com/117535328/205509395-2fee259a-988c-4555-a110-dd8f23996922.jpg)

**Comment by proudx2** [comment]
Do any other of your ir devices work?  What is the device you are trying to control. You might want to download a custom .bin for the device and select custom and put in code for it.

Sent from my iPhone

On Dec 3, 2022, at 7:24 AM, edutsilva ***@***.***> wrote:

﻿

Thank for replying. The sbav-jb,sav is in C:\Program Files (x86)\Sling Media\SlingPlayer\SBAV
I am from Brazil. Like I said, the remote control is working for my other box (M1) so It worked for the ProHd before (using slinger) it got bricked. So chances are that the unbrickig caused this issue.
When I try to use the Slingplayer APP, I can locate what is supposed to be the right controller, but it does not work. I checked the infrared emitter with my cell phone and it is sending the commands to the cable box, but it is not responding.
Any ideias?

**Reply by socelillyhammer** [reply]
do you know if there is a custom .bin for the fios one tv mini boxes?

**Reply by edutsilva** [reply]
I´m trying to control a cable box from a Brazillian company called NET BRAZIL. As I said, my other slingbox (M1) is connected to another cable box from the same company and the remote control works fine. So I imagine slinger has all it needs to handle these IR codes. 
Where can I find the .bin file? Thanx.

**Reply by edutsilva** [reply]
My remote control problema is solved. I found out my cable box manufacturer and found the right IR configuration using Slingplayer.
Thank you for trying to help.

**Comment by socelillyhammer** [comment]
What steps are you doing to properly protect? I.e. what ports or ip address
are you blocking? I can't seem to find the exact numbers anywhere.

**Reply by bradut** [reply]
@socelillyhammer I've blocked on my router the ports **5678, 5680, 3724, 3490, 3074** and didn't get bricked again in the past 24 hours since I unbricked my PRO-HD. 
Some suggestions indicated to also block 80, 8080, 443, 
Also, I didn't block any IP.

How to block the ports depends on the type or the firmware of the router. Mine is re-flashed with DD-WRT.
I think we need a channel dedicated to port blocking either here or on Facebook or on Discord

**Reply by proudx2** [reply]
The guide list all outbound destination ports the prohd uses.  Bricking appears to come from port 5678. Blocking just  port 5678 outbound from slingbox will stop bricking

**Reply by sp222222** [reply]
> @socelillyhammer I've blocked on my router the ports **5678, 5680, 3724, 3490, 3074** and didn't get bricked again in the past 24 hours since I unbricked my PRO-HD. Some suggestions indicated to also block 80, 8080, 443, Also, I didn't block any IP.
> 
> How to block the ports depends on the type or the firmware of the router. Mine is re-flashed with DD-WRT. I think we need a channel dedicated to port blocking either here or on Facebook or on Discord

there is a port block on discord under general on the right side

dont block 80 8080 443 as a whole.

**Comment by edutsilva** [comment]
Your IP won't change. As long a you don't change it. You will not be using
your WAN ip, which is dynamic. You are using your lan IP.
You are still using schedule "never". Nothing will work if it scheduled to
NEVER work.


Em seg., 5 de dez. de 2022 14:33, PeteSki1618 ***@***.***>
escreveu:

**Reply by PeteSki1618** [reply]
Ok thanks.  So leave it as is in the screenshot except change it to Always, not Never, right?  Or do I have to change the Destination IP Range to LAN as well?  Thanks again for your prompt reply.

**Comment by edutsilva** [comment]
Just change to always. And try to unbrick the sb.

Em seg., 5 de dez. de 2022 14:58, PeteSki1618 ***@***.***>
escreveu:

**Reply by PeteSki1618** [reply]
Instantly bricked it.  Ugh!  On to plan B or should I say Q.  Been trying this for over a month.

**Comment by socelillyhammer** [comment]
Please let me know what you did as I'm trying to do this on the SB 500 with
the verizon arris ipc4100 for the IR remote.  Thank you.

**Reply by sp222222** [reply]
socelilly —- STOP hijacking threads ! this is the proHd discussion. Not 500

**Comment by edutsilva** [comment]
I had to set up the remote control of my now unbricked ProHD slingbox.
Since it is an old client (1.5), it did not have the right remote of my
cable box company, but it had the right one when I used the manufacturer
and the model to search for. Mine is a Pace. Once you find the right
controller inside the SlingPlayer and your device responds the IR commands,
just finish the configuration and restart slinger.

Em seg., 5 de dez. de 2022 15:23, socelillyhammer ***@***.***>
escreveu:

> Please let me know what you did as I'm trying to do this on the SB 500 with
> the verizon arris ipc4100 for the IR remote. Thank you.
>

**Comment by socelillyhammer** [comment]
Sorry but i am not following.  Would you mine providing the steps on what
to do. i.e. step 1 open the config file. step 2. go to "xyz" etc.  Thank you

**Comment by edutsilva** [comment]
Is your slingbox bricked? Because the procedure to unbrick the ProHD may
not work for your model.
I really don't know. If it does work for your model, all you have to do is
follow the instucions you will find here in this thread. First post.
During the procedure you will have the chance to select the right remote
control.
Good luck.

Em seg., 5 de dez. de 2022 15:53, socelillyhammer ***@***.***>
escreveu:

> Sorry but i am not following. Would you mine providing the steps on what
> to do. i.e. step 1 open the config file. step 2. go to "xyz" etc. Thank you
>

**Comment by swilsonnc** [comment]
Also, my ProHD, while unbricked, still had weird issues with slinger (errors).  The firmware on my ProHD was 1.1.250 and I updated it to 1.5.170 using Slingplayer 1.5 and it is working great now.  Just something to keep in mind.
